pkg/inject/inject.go

```diff
@@ -539,6 +548,36 @@ func (conf *ResourceConfig) serviceAccountVolumeMount() *corev1.VolumeMount {
 	return nil
 }
 
+func (conf *ResourceConfig) trace() *charts.Trace {
+	var (
+		svcAddr    = conf.getOverride(k8s.ProxyTraceCollectorSvcAddr)
```

Comment:
just curious why these are var definitions instead of `:=` assignments

Reply:
Just a habit of grouping variables together.